components/styleguide/form-items/field-label-states/_field-label-states.css

```diff
@@ -0,0 +1,34 @@
+/*
+  Field Label States
+
+  Weight: 0
+
+  Markup: field-label-states.html.twig
+
+  Style guide: styleguide.form_items.field_label_states
+*/
+
+
+
+.field--label-inline .field__label {
+  float: left;
+  padding-right: 0.5rem;
+}
+
+.field--label-inline .field__label {
+  float: left;
+  padding-right: 0.5rem;
+}
+
+.field--label-inline .field__label:after {
+  content: ":";
+}
+
+.field--label-visually_hidden .field__label.visually_hidden {
```

Comment:
Could we have the visually hidden added as a component so we don't have to declare every element with this property? I mean like this: http://cgit.drupalcode.org/zen/tree/STARTERKIT/components/init

Reply:
I'm not sure exactly what you mean here 'to add it as a component'. 

It's already in Drupal so we won't need that file at all in our theme libraries, but we need to have it in the styleguide so we can replicate what is going to happen when Drupal renders the HTML/classes..